.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: node_js
+node_js:
+- 10
+install:
+- npm install
+script:
+- npm run build
+- npm run build-prod
+deploy:
+  provider: pages
+  local_dir: src
```

Comment:
Shouldn't this be `dist`? See https://angular.io/guide/deployment#basic-deployment-to-a-remote-server (and the gh-pages section below it might be relevant too)